include/pajlada/signals/connection.hpp

```diff
@@ -202,7 +202,7 @@ class Connection
     }
 
     bool
-    block()
+    block() const
```

Comment:
While this and unblock technically works as const, they feel weird to me since the connection body function they're calling is not const. Do you have a specific need for these to be const?

Reply:
No, I don't have a need for this.